docs/dev/tools/cli.md

```diff
@@ -1191,45 +1244,42 @@ Examples:
 viam machines list
 
 # get machine status
-viam machines status  --machine=123 --organization="Robot's Org" --location=myLoc
+viam machines status  --machine=123 --location=myLoc
```

Comment:
I'm a little unclear about why we don't include `organization` anymore here, but we include it immediately above? Is the goal for one to be all flags and the other to be only mandatory flags? If so, we can probably get rid of `location` here as well.

Reply:
Yeah that was how I was writing this. Oops yes will remove location.